x/auth/ante/sigverify_test.go

```diff
@@ -100,8 +100,9 @@ func (suite *AnteTestSuite) TestConsumeSignatureVerificationGas() {
 	}
 	for _, tt := range tests {
 		sigV2 := signing.SignatureV2{
-			PubKey: tt.args.pubkey,
-			Data:   tt.args.sig,
+			PubKey:          tt.args.pubkey,
+			Data:            tt.args.sig,
+			AccountSequence: 2, // Arbitrary account sequence
```

Comment:
Ah, there is the magic 2 I was hunting above.

If this is used in lots of test setup, can you pull it out to a public constant that can be imported as needed in other tests?

Reply:
Makes sense. What I did: when it's arbitrary, but used in other places in tests, I assigned them to a variable. If it's really arbitrary, as in doesn't influence the tests whatsover, I just put 0.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/6997#discussion_r470629273